saleor/graphql/webhook/types.py

```diff
@@ -45,3 +109,7 @@ class Meta:
     @staticmethod
     def resolve_events(root: models.Webhook, *_args, **_kwargs):
         return root.events.all()
+
+    @staticmethod
+    def resolve_deliveries(root: models.Webhook, *_args, **_kwargs):
+        return core_models.EventDelivery.objects.filter(webhook_id=root.pk)
```

Comment:
We need to make sure that we return deliveries sorted by creation date in descending order. Do we have any default sorting in the model's meta class? It might be a good place to add it so we have it by default everywhere.

Reply:
The same applies to EventDeliveryAttempt model.